x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list_cell_renderer.tsx

```diff
@@ -36,6 +37,10 @@ export const RuleEventLogListCellRenderer = (props: RuleEventLogListCellRenderer
     return null;
   }
 
+  if (columnId === 'message') {
+    return <EuiText size="xs">{value}</EuiText>;
```

Comment:
It doesn't look like the flex group is aligning center.  Let's just change this to `size='s'` instead.  Apologies for the back and forth on this.  

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/3756330/180792241-e1dfa033-53e6-4350-8423-ba27d3605e3e.png">